### PR TITLE
Remove errors and warnings of Javadocs

### LINKED ISSUE #13: Javadocs cannot be generated
I tried to make javados according to README but the following command made many errors.

```
./gradlew javadoc
```

### PR DESCRIPTION
**Pull request by tiwanari**
This will close #13. 

UPDATE:
I found a bug in `Servomotor` [here](https://github.com/PileProject/drivecommand/pull/15/files#diff-ae258c6fbe84a4ffb350d84b373a4982R58) and fixed in this PR (I didn't intend to include this, sorry).

### CODE REVIEW

File: src/main/java/com/pileproject/drivecommand/command/CommandBase.java [hunk 1]
```diff
@@ -32,27 +35,28 @@ public CommandBase(CommandType type, Map<String, Object> args) {
     }
 
     /**
-     * Get its command type
+     * Get this command type.
```
Comment (mandaiy) [inline src/main/java/com/pileproject/drivecommand/command/CommandBase.java:38]: General comment:
Javadoc comment for methods are usually start with third-person singular verb
http://www-inf.int-evry.fr/cours/java/JAVADOC/writingdoccomments.html
example: https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html

**Review by mandaiy — COMMENTED** [review]
You did not mention _what was the problem_ and _what this commit solved_.
Why did you encounter the error you noted on #13 and how did you solve that?

**Comment by tiwanari** [comment]
Sorry, let me explain.

# Problem

Javadocs have some errors and warnings ( #13 ).

## Errors

Errors occurred because some `@param`s didn't have the corresponding names like this;

``` java
/**
 * @param int number a number to be used for ...
 */
public void foo(int number) {
   // ...
}
```

We should put the name of argument right after `@param` so this should be like this;

``` java
/**
 * @param number a number to be used for ...
 */
public void foo(int number) {
   // ...
}
```

There were relatively not so many errors but they prevented us from making Javadocs.

## Warnings

There were many warnings for Javadocs;
- No explanation for `@param`
- No explanation for `@return`
- No explanation for `@throws`

# What this PR solves

I fixed the above problems by rewriting them or adding some more texts. 
In addition to that, I checked all Javadocs in order to keep Javadocs' consistency.

File: src/main/java/com/pileproject/drivecommand/command/CommandBase.java (hunk 1 above)
Comment (tiwanari) [reply src/main/java/com/pileproject/drivecommand/command/CommandBase.java:38]: Thanks! The document is very helpful.

Comment (mandaiy) [reply src/main/java/com/pileproject/drivecommand/command/CommandBase.java:38]: Maybe we need some agreement on this and other developing conventions.
These should be shared over our team and noted on a wiki system or a website.

Comment (tiwanari) [reply src/main/java/com/pileproject/drivecommand/command/CommandBase.java:38]: Yeah, you are right. So, how do we keep them? Do you want me to make a wiki or something?

Comment (mandaiy) [reply src/main/java/com/pileproject/drivecommand/command/CommandBase.java:38]: Wiki seems good to me for this purpose. Let's discuss on slack.

**Review by mandaiy — APPROVED** [review]
Do you plan to make changes on this? Otherwise you can merge this and let's fix docs later.

**Comment by tiwanari** [comment]
Thanks for your review. 
I was going to rewrite some docs according to the link you provided. But it will take a substantial time and now I want to fix the current situation in which users can't see any documents at all. So I'll close this and make an issue as a note.

**Pull request merged**